src/app/forms/custom-sections/psv-brakes/psv-brakes.component.ts

```diff
@@ -57,6 +57,7 @@ export class PsvBrakesComponent implements OnInit, OnChanges, OnDestroy {
       .subscribe(([selectedBrake, event]) => {
         // Set the brake details automatically based selection
         if (selectedBrake && event?.techRecord_brakes_brakeCodeOriginal) {
```

Comment:
Should we change this check as it doesn't appear that we're using `event?.techRecord_brakes_brakeCodeOriginal` any more?

Reply:
@naathanbrown, @gjulien-bjss  ^^ ?